src/sage/combinat/designs/covering_array.py

```diff
@@ -0,0 +1,43 @@
+r"""
+Covering Arrays (CA)
+
+A Covering Array, denoted CA`(N,k,v,t)`, is an `n` by `k` array with
```

Comment:
This definition is weirdly formulated : each row is unlikely to contain every sequence of `t`-elements.